src/fdcache.cpp

```diff
@@ -414,6 +414,7 @@ FILE* FdManager::MakeTempFile() {
     strncpy(cfn, fn.c_str(), sizeof(cfn) - 1);
     cfn[sizeof(cfn) - 1] = '\0';
 
+    umask(0600);
```

Comment:
I think mkstemp creates a file with attributes of 0600, so why is the umask being used to enforce the attributes here?
Also, if you are setting a umask, I think you need to set it back to the original mask value before exiting this function.

Reply:
Reverted.  Coverity give bad advice here as described by: https://github.com/cockpit-project/cockpit/issues/760